android/app/src/test/kotlin/net/mullvad/mullvadvpn/viewmodel/LoginViewModelTest.kt

```diff
@@ -86,6 +88,8 @@ class LoginViewModelTest {
             val uiStates = loginViewModel.uiState.testIn(backgroundScope)
             val sideEffects = loginViewModel.uiSideEffect.testIn(backgroundScope)
             coEvery { mockedAccountRepository.login(any()) } returns LoginResult.Ok
+            coEvery { mockedAccountRepository.accountExpiryState } returns
```

Comment:
We should perhaps not use `DateTime.now()` in a test and use static time instead?

Reply:
How would you do it instead? We need to ensure the time is in the future, I was thinking this was the way to go that won't fail over time?

Reply:
I guess you could use a mock instead if not too much work?